testing/preview_all_cameras/preview_all_cameras.py

```diff
@@ -0,0 +1,27 @@
+#Example for visualization of all cameras from one device
+import cv2
+import depthai
+from depthai import node, message
+
+class all_cams(depthai.Node):
+    def __run__(self, right_stereo: message.ImgFrame,left_stereo: message.ImgFrame,color: message.ImgFrame):
+        #convert to opencv frames
+        right_stereo_cv = right_stereo.getCvFrame()
+        left_stereo_cv = left_stereo.getCvFrame()
+        color_cv = color.getCvFrame()
+
+        #Show image frames
+        cv2.imshow("right_stereo",right_stereo_cv)
+        cv2.imshow("left_stereo", left_stereo_cv)
+        cv2.imshow("color", color_cv)
```

Comment:
this looks like a lot of duplication.

Reply:
My goal was to keep it as simple as possible even when it mean to have little bit of repetition. Bud I do agree that writing it in more correct way would help in more complex extensions of example of user so I rewritten it